extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php

```diff
@@ -116,133 +107,126 @@ public function renderInfobox( array $infoboxdata, $theme, $layout ) {
 	 *
 	 * @return string - group HTML markup
 	 */
-	private function renderGroup( $groupData ) {
+	protected function renderGroup( $groupData ) {
 		$cssClasses = [ ];
-		$helper = new PortableInfoboxRenderServiceHelper();
 		$groupHTMLContent = '';
-		$dataItems = $groupData[ 'value' ];
-		$layout = $groupData[ 'layout' ];
-		$collapse = $groupData[ 'collapse' ];
+		$dataItems = $groupData['value'];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "value" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)